libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -764,8 +764,15 @@ void YamlConfiguration::parseSingleProperty(const std::string& propertyName, con
   core::Property myProp(propertyName, "", "");
   processor->getProperty(propertyName, myProp);
   const PropertyValue coercedValue = getValidatedProcessorPropertyForDefaultTypeInfo(myProp, propertyValueNode);
```

Comment:
what about this `getValidatedProcessorPropertyForDefaultTypeInfo` function? it seems to log errors on exception, in which case does it not log, when it should?

Reply:
That function does the cast through `yaml-cpp` lib's `YAML::Node` class and it does not run our custom validators for those properties, those are done through the `PropertyValue` class.

Reply:
oh now I see, so even though `PropertyValue` contains a `validator_`, assignment in itself does not execute it, it only happens in `setValue` (which is called from `processor->setProperty`)